shared/src/libs/packages/map/libs/interfaces/map.interface.ts

```diff
@@ -0,0 +1,13 @@
+interface IMapService {
```

Comment:
do you need this interface in shared folder ?

Reply:
move it to the frontend, because I suppose it will be used only on fe. If we will need it on shared/be - then we will move it. Storage is in shared, because in future it can be reused not only in fe. With maps I think the situation will be different